libnavigation-base/src/main/java/com/mapbox/navigation/base/options/NavigationOptions.kt

```diff
@@ -41,6 +41,7 @@ const val DEFAULT_NAVIGATOR_PREDICTION_MILLIS = 1000L
  * @param historyRecorderOptions defines configuration for recording navigation events
  * @param eventsAppMetadata [EventsAppMetadata] information (optional)
  * @param enableSensors enables sensors for current position calculation (optional)
+ * @param isCopilotFeedbackOnlyEnabled true if Copilot History files should be sent only when they include Feedback events. Default is false
```

Comment:
I don't like `enabled` here. You don't really enable anything I guess. Also from the name I thought it was more like "send only copilot feedback". But my suggestion would be sth like `shouldSendCopilotEventsOnlyWithFeedback` which is probably too long. Haven't you thought about introducing a `CopilotOptions` parameter instead of a boolean flag? It would allow us to include more Copilot stuff later and we'd also be able to make names shorter (we can remove at least Copilot from the name).